Week3/homework/MY-step2-1-Hello_I_am_bar.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+/*sample*/
+// function doIt() {
+//   console.log('I am done');
+// }
+// setTimeout(doIt, 5000);
+
+//Test this code by copy-pasting the whole foo and bar functions to the browser console
+function foo(func) {
+  // What to do here?
+  // Replace this comment and the next line with your code
+  return ('Hi, I am AaaaT, who are you?');
```

Comment:
Same problem again. This function should take a function and invoke/call it. Not return anything.